spec/e2e/accessibility.test.js

```diff
@@ -0,0 +1,62 @@
+import { describe, before, after, test, it } from 'node:test';
+import assert from 'node:assert';
+import { relative, dirname } from 'node:path';
+import glob from 'fast-glob';
+import puppeteer from 'puppeteer';
+import { AxePuppeteer } from '@axe-core/puppeteer';
+import { createServer } from 'http';
+import sitepath from 'path';
+import handler from 'serve-handler';
+import getPort from 'get-port';
+
+const paths = glob
+  .sync('_site/**/index.html')
+  .map((path) => dirname(relative('_site', path)))
+  .filter((path) => path !== '.')
+  .map((path) => `/${path}/`);
+
+describe('accessibility', () => {
+  /** @type {string} */
+  let publicPath;
+
+  /** @type {Server} */
+  let server;
+
+  /** @type {number} */
+  let port;
+
+  /** @type {import('puppeteer').Browser} */
+  let browser;
+
+  before(async () => {
+    publicPath = sitepath.relative(process.cwd(), sitepath.resolve('../../_site'));
+    port = await getPort();
+    browser = await puppeteer.launch();
+    server = createServer((request, response) =>
+      handler(request, response, { public: publicPath }),
```

Comment:
I'm trying to follow the logic for `publicPath` and wondering if it's even necessary to have at all, vs. just assuming the tests are run from the root directory.

```suggestion
      handler(request, response, { public: '_site' }),
```